### PR TITLE
declare python3 support in trove and wheel

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
@zedlander when deploying you'll need to run:

`python3 setup.py sdist bdist_wheel register upload`

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```
Comment (adamchainz) [inline setup.cfg]: `[wheel]` is the new header

Comment (graingert) [inline setup.cfg]: `[wheel]` is legacy https://bitbucket.org/pypa/wheel/src/4da780b849affbff98a0defb21ad1f30a94d6f57/wheel/bdist_wheel.py?at=default&fileviewer=file-view-default#bdist_wheel.py-119

Comment (adamchainz) [inline setup.cfg]: Oh thx

**Comment by RonnyPfannschmidt** [comment]
perhaps update `release.sh` as well?

**Comment by adamchainz** [comment]
@graingert ping I want to see this wheel'd up :)

**Comment by graingert** [comment]
@zedlander ping

**Comment by adamchainz** [comment]
@graingert update release.sh?

**Pull request merged**